ajax/libs/jodit/package.json

```diff
@@ -16,12 +16,26 @@
     "type": "git",
     "url": "https://github.com/xdan/jodit.git"
   },
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/xdan/jodit.git",
```

Comment:
Let's remove git auto-update config part.
As you can see, the CI said `should not have multiple auto-update configs`, so we just use npm auto-update. Thank you.